src/main/webapp/WEB-INF/views/manager/edit-booking.jsp

```diff
@@ -35,6 +30,11 @@
                 </ul>
             </nav>
         </div>
+        <div id="create-booking-btn">
+            <button class="btn btn-primary" onclick="openCreateBookingDialog()">
+                Add a kid
```

Comment:
Thank for advice. You are right. I saw this problem, but I made a decision don't fix it. Because I didn't create this part of code. I just move it from one place to another, because there was a problem with tabulation. Moreover my bug is about tabulation. So this problem must resolve by somebody else.